adr/0015-home-assistant-one.md

```diff
@@ -0,0 +1,82 @@
+# 0015. Installation method: Home Assistant One
+
+Date: 2020-06-08
+
+## Status
+
+Accepted
+
+## Context
+
+Define a supported installation method as per [ADR-0012](https://github.com/home-assistant/architecture/blob/master/adr/0012-define-supported-installation-method.md).
+
+## Decision
+
+Home Assistant One is the full installation of our all-inclusive home automation system. Best in class home automation is complemented with a UI for configuring your system, making backups and safe updates with automatic rollback.
+
+This is the general recommended installation method and the one that our website and documentation should be focused on.
+
+### Supported boards/hardware/machines
+
+- Raspberry Pi 2
+- Raspberry Pi 3 Model B and B+ 32bit
+- Raspberry Pi 3 Model B and B+ 64bit
+- Raspberry Pi 4 Model B 32bit
+- Raspberry Pi 4 Model B 64bit
+- Tinkerboard
+- ODROID-C2
+- ODROID-N2
+- ODROID-XU4
+- Intel NUC
+- Virtual Machine (64 bits intel-based)
+
+### Supported Operating Systems and versions
+
+This installation method is only available and supported using the Home Assistant Operating System. Only the latest major version is supported (Currently, 4.x).
+
+When a new major version is released, the previous major version will be dropped with a deprecation period of 2 months. The last 3 minor releases are supported.
+
+### Supported Hypervisors
+
+The Home Assistant Operating System can be run on a Hypervisor and thus be run as a virtual machine. The following Hypervisors are supported:
+
+- TBD. Maybe:
+- VirtualBox
+- Proxmox
+- UnRaid
+- VMWare ESX
+- VMWare Workstation
+- HyperV
```

Comment:
I know the list is TBD, but Hyper-V does not have USB passthrough, due to that there are a few things that will not work.
I do not feel that it should be a part of supported hypervisors for that reason alone.

Reply:
Good point, agree on that.

Reply:
Let's rename it to Xen